Framework/API/src/MatrixWorkspace.cpp

```diff
@@ -70,7 +69,6 @@ MatrixWorkspace::MatrixWorkspace(const MatrixWorkspace &other)
   m_isInitialized = other.m_isInitialized;
```

Comment:
I'm probably missing something here but what happens to `m_isCommonBinsFlagValid` here?

Reply:
It looks like it. I think it should be copied over? Although the worst case is it just recomputes I guess...